api/capi/include/nnstreamer-single.h

```diff
@@ -106,6 +106,26 @@ int ml_single_close (ml_single_h single);
  */
 int ml_single_invoke (ml_single_h single, const ml_tensors_data_h input, ml_tensors_data_h *output);
 
+/**
+ * @brief Invokes the model with the given input data with the given info.
+ * @details This function changes the input data dimensions for the model.
+ *          Note that this has a default timeout of 3 seconds. If an application wants to change the time to wait for an output, set the timeout using ml_single_set_timeout().
+ * @param[in] single The model handle to be inferred.
```

Comment:
@myungjoo @again4you how should `@since_tizen` be described here? I have currently omitted it.

Reply:
It'd be ```@since_tizen 6.0```

Reply:
updated